pkgs/development/python-modules/elastic-transport/default.nix

```diff
@@ -77,6 +77,7 @@ buildPythonPackage rec {
     "test_debug_logging_no_body"
     "test_httpbin"
     "test_sniffed_nodes_added_to_pool"
+    "test_async_transport_httpbin"
```

Comment:
This test accesses `httpbin.org` on the network, which isn’t okay and would break if Hydra had the sandbox enabled. We already disable `test_httpbin`, which is the synchronous version of the test.